src/components/Tabs.tsx

```diff
@@ -30,6 +30,8 @@ export interface TabsProps<V extends string, X> {
   tabs: Tab<V>[];
   onChange: (value: V) => void;
   contentXss?: X;
+  // Allow for showing the tabs even if there is only one enabled tab
+  forceShow?: boolean;
```

Comment:
Wdyt about `hideIfOnlyOneTabActive` and it defaults to `true`?

Reply:
Personally I prefer boolean/flag type props to be enabled by just passing them with a value, something like:
`<Tabs forceShow />` vs `<Tabs hideIfOnlyOneTabActive={false} />`.
I could go for a prop like: `disableHideTabs`, thoughts?

Reply:
Ah yeah, I guess... `disableHideTabs` has a double negative, maybe `showAllTabsEvenIfOnlyOneIsActive`? :-) 

I dunno, given the core feature is "hiding the one tab" and it defaults to "on", it's hard to avoid a double-negative of "don't hide" in the name to achieve the "setting to true" --> "inverts the default ... which is also true".

So, dunno, still kinda like the `=false` --> turning off the feature, but maybe some form of `showAllTabs`, `alwaysShowAllTabs`...